apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -178,4 +178,6 @@
     public static final OfficialComponent QUARTZ_SCHEDULER = new OfficialComponent(97, "quartz-scheduler");
 
     public static final OfficialComponent XXL_JOB = new OfficialComponent(98, "xxl-job");
-}
+
+    public static final OfficialComponent SPRING_WEBCLIENT = new OfficialComponent(99, "spring-webflux-5.x-webclient");
```

Comment:
A component should not include the version.

Reply:
ok,I wiil modefy it after this CI finished.